src/VsixV3/ProjectTemplates1.x/Microsoft.VisualStudio.NetCore.ProjectTemplates.1.x.swr

```diff
@@ -0,0 +1,16 @@
+use vs
+
+package name=Microsoft.VisualStudio.NetCore.ProjectTemplates.1.x
+        version=$(Version)
+
+vs.localizedResources
+  vs.localizedResource language=en-us
+                       title="Microsoft VisualStudio Managed Project Templates NetCore 1.x"
+                       description="Microsoft VisualStudio Managed Project Templates for NetCore 1.x Projects"
+
+folder "InstallDir:Common7\IDE\Extensions\Microsoft\Dotnet"
+  file source="$(SourcePath)\VsixV3\ProjectTemplates1.x\Templates.pkgdef"
+  
+folder "InstallDir:Common7\IDE\Extensions\Microsoft\Dotnet\Templates\NetCore\1.x"
+  file source="$(OutputPath)\microsoft.dotnet.common.projecttemplates.1.x.$(ProjectTemplateVersion).nupkg"
```

Comment:
Should the deployed file have the version? The web templates strip out the version - is that not necessary?